### DISCUSSION TITLE
google authorizationCode

### DISCUSSION THREAD

**Discussion by havart** [abacritt/angularx-social-login#421] [Q&A]
Hello, 
Could you please help me understand where I can get the authorization code.
It is not in the response singIn.
![image](https://user-images.githubusercontent.com/12501647/123391084-50177700-d5a4-11eb-8999-bc7756a9f8cf.png)

My config: 
`googleLoginOptions = {
    scope: 'profile https://www.googleapis.com/auth/content',
    response_type: 'id_token permission code',
};`

**Comment by carlosempd** [comment]
Hi @havart did you find a workaround to get the authorization code?
